docs/about.html

```diff
@@ -5,16 +5,13 @@
         <meta charset="UTF-8" />
         <meta name="author" content="Felipe Bassi" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="description" content="The PSU Behrend DIGIT site, made by DIGIT students for the interested. DIGIT course info, events, student projects, and a few galleries." />
```

Comment:
@WillStill This is terrific! Thank you! It *should* make the page more accessible for web crawlers and indexers.

Reply:
@WillStill Think of super specific words that help cover the different kinds of content that students work on / that is featured here.